CHANGELOG.md

```diff
@@ -41,6 +41,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ### Changed
 
 - Updated `example/cumulus-tf/variables.tf` to have `cmr_oauth_provider` default to `launchpad`
+- Pinned @aws-sdk v3 packages to ~3.186.0 to address syntax issues
+- Upgraded saml2-js from 3.1.0 to 4.0.0
```

Comment:
Maybe we can add ticket number now https://bugs.earthdata.nasa.gov/browse/CUMULUS-3104